index.js

```diff
@@ -334,5 +344,192 @@ module.exports = {
             '@typescript-eslint/no-empty-interface': 'off',
          },
       },
+      {
+         files: [ '**/*.vue' ],
+         parser: 'vue-eslint-parser',
+         parserOptions: {
+            parser: '@typescript-eslint/parser',
+            ecmaVersion: 2020,
+            sourceType: 'module',
+         },
+         env: {
+            browser: true,
+         },
+         rules: Object.assign(
```

Comment:
I ran these rules against one of our projects and I'm seeing these errors for every file:

```
1:1   error  Definition for rule 'vue/no-deprecated-v-is' was not found             vue/no-deprecated-v-is
1:1   error  Definition for rule 'vue/no-invalid-model-keys' was not found          vue/no-invalid-model-keys
1:1   error  Definition for rule 'vue/no-this-in-before-route-enter' was not found  vue/no-this-in-before-route-enter
1:1   error  Definition for rule 'vue/require-emit-validator' was not found         vue/require-emit-validator
```

All of the other rules seem to be executing fine. Any idea why? FWIW, I used `npm link` to test so maybe that's causing a problem somehow?

Reply:
Can we disable the `vue/no-multiple-template-root` rule? I think it's "on" by default, but multiple template roots are a new feature in Vue 3, are there are legitimate use cases for it.